src/guards.js

```diff
@@ -105,4 +105,22 @@ export const isAskedAboutUsernameDiamond = (text) => {
  */
 export const isAskedWhoMadeMe = (text) => {
     return /who(?:\s+(?:are|is) your)?\s+(?:made|created|owns|develop(?:s|ed|ers?)|maintain(?:s|ers?))(\s+you)?/.test(text);
+};
+
+/**
+ * @summary checks if the message asked for candidate score of another user
+ * @param {string} text
+ * @returns {boolean}
+ */
+export const isAskedForOtherScore = (text) => {
+    return /what(?: i|')s(?: the)? candidate score (?:for|of) \d+(?:$|\?)/.test(text);
```

Comment:
Regex for this compared to the guard `isAskedForCandidateScore` is slightly different, allowing a trailing `?`. We should also update `isAskedForCandidateScore` to allow a trailing `?`.

What about allowing a URL of the user's site profile link instead of just ID? Do you think it's possible to add to this regex?
e.g.: `what is the candidate score for https://stackoverflow.com/users/584192/samuel-liew`

Reply:
Yup, this is something I only came up yesterday by noticing that the command matching was a bit too restrictive by not allowing trailing question marks at the end (a thing, I think, most chat users won't pay too much attention to) - probably needs to be added to the rest of the guards - will do when I have some time!

Re: regex - yup, sounds like a good idea to me - will need some refactoring on the part of the message builder, but will be nice to have (I thought about requesting candidate score by name too, but that might turn unnecessarily complicated)